nano/core_test/peer_container.cpp

```diff
@@ -11,71 +19,100 @@ TEST (peer_container, empty_peers)
 	ASSERT_EQ (0, network.size ());
 }
 
-TEST (peer_container, no_recontact)
```

Comment:
This look like an important test to me that should not be removed.
I think a node should not get into a reconnect loop, if another node is refusing it service.
Which is what I think this test is testing.
So, let's at least raise an issue too re-instate it in the future.

Reply:
This test is testing something that doesn't exist in our TCP implementation. This is why it was removed.